comptests/prov/test_prov_k8s_cortx.py

```diff
@@ -431,3 +441,94 @@ def test_32654(self):
+                                        (self.deploy_cfg['iface']))
+        LOGGER.info(resp)
+        eth1_ip = resp[1].strip("'\\n'b'")
+        LOGGER.info("IP of ethernet 1: %s ",eth1_ip)
+        if self.service_type == "NodePort":
+            resp = ext_lbconfig_utils.configure_nodeport_lb(self.master_node_list[0],
+                                                            self.deploy_cfg['iface'])
+            LOGGER.info(resp)
+            if not resp[0]:
+                LOGGER.debug("Did not get expected response: %s", resp)
+                # assert_utils.assert_true(resp[0], "False")
+            ext_ip = resp[1]
+            LOGGER.info("External IP: %s", ext_ip)
+            port = resp[2]
+            http_port = resp[3]
+            ext_port_ip = self.deploy_cfg['https_protocol'].format(ext_ip)+\
+                                  ":{}".format(port)
+            LOGGER.debug("External LB value, ip and port will be: %s", ext_port_ip)
+        else:
+            LOGGER.info("Configure HAproxy on client")
+            ext_lbconfig_utils.configure_haproxy_rgwlb(self.master_node_list[0].hostname,
+                                                       self.master_node_list[0].username,
+                                                    self.master_node_list[0].password,
+                                                            eth1_ip, self.deploy_cfg['iface'])
+            ext_port_ip = self.deploy_cfg['https_protocol'].format(eth1_ip)
+        LOGGER.info("Step 3:Configure AWS on Client")
+        resp = self.master_node_obj.execute_cmd(cmd=commands.AWS_INSTALL)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info(resp)
+        resp = self.master_node_obj.execute_cmd(cmd=commands.AWS_PLUGIN)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info(resp)
+        resp = self.master_node_obj.execute_cmd(cmd=commands.AWS_VERSION)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info(resp)
+        resp = self.master_node_obj.execute_cmd(cmd=commands.CMD_AWSCLI_CONF + " set plugins.endpoint awscli_plugin_endpoint")
+        LOGGER.info(resp)
+        resp = self.master_node_obj.execute_cmd(cmd=commands.CMD_AWSCLI_CONF + " set default.region us-east-1")
+        LOGGER.info(resp)
+        resp = self.master_node_obj.execute_cmd(cmd=commands.CMD_AWSCLI_CONF + " set aws_access_key_id {}".format(access_key),
+                                                   read_lines=True)                         
+        resp = self.master_node_obj.execute_cmd(cmd=commands.CMD_AWSCLI_CONF + " set aws_secret_access_key {}".format(secret_key),
+                                                    read_lines=True)
+        LOGGER.info("Step 4:Creating bucket")
+        resp = self.master_node_obj.execute_cmd(cmd=commands.CMD_AWSCLI_CREATE_BUCKET.format(bucket_name) + " --endpoint-url " +  (self.prov_deploy_cfg["endpoint_url"]).format(http_port),
```

Comment:
Is there any explicit need to execute s3 operation from master node. 
S3 test lib can be used to perform all the IO operations from client. 
@aayushisharma88 Please let me know if there is any such requirement.